Formula/git-recent.rb

```diff
@@ -0,0 +1,16 @@
+class GitRecent < Formula
+  desc "See your latest local git branches, formatted real fancy"
+  homepage "https://github.com/paulirish/git-recent"
+  url "https://raw.githubusercontent.com/paulirish/git-recent/029cc001030d224c961da95502ecee0b54af1989/git-recent"
+  version "1.0.3"
+  sha256 "796dce33edcf6b6031aa0729e443fd4ec46314809bcf3d425d895c90ffe50833"
+
+  def install
+    bin.install "git-recent"
+  end
+
+  test do
+    system "git", "init"
+    system "git", "recent"
```

Comment:
Yep, sounds good. An `assert_match` any non-error, non-empty partial match would be perfect.